google/cloud/iam/quickstart/README.md

```diff
@@ -0,0 +1,167 @@
+# HOWTO: using the Cloud IAM C++ client in your project
+
+This directory contains small examples showing how to use the Cloud IAM C++
+client library in your own project. These instructions assume that you have
+some experience as a C++ developer and that you have a working C++ toolchain
+(compiler, linker, etc.) installed on your platform.
+
+* Packaging maintainers or developers that prefer to install the library in a
+  fixed directory (such as `/usr/local` or `/opt`) should consult the
+  [packaging guide](/doc/packaging.md).
+* Developers wanting to use the libraries as part of a larger CMake or Bazel
+  project should consult the current document. Note that there are similar
+  documents for each library in their corresponding directories.
+* Developers wanting to compile the library just to run some of the examples or
+  tests should consult the
+  [building and installing](/README.md#building-and-installing) section of the
+  top-level README file.
+* Contributors and developers to `google-cloud-cpp` should consult the guide to
+  [setup a development workstation][howto-setup-dev-workstation].
+
+[howto-setup-dev-workstation]: /doc/contributor/howto-guide-setup-development-workstation.md
+
+## Before you Begin
+
+To run the quickstart program you will need a working Google Cloud Platform
+(GCP) project, and list the existing service accounts for your project.
+The [Cloud IAM quickstarts][iam-quickstart-link] is a good place to find
+information on how to setup a GCP project.
+
+## Configuring authentication for the C++ Client Library
+
+Like most Google Cloud Platform (GCP) services, Cloud IAM requires that
```

Comment:
nit: we already defined GCP, no need to define it again (probably inherited this problem from all the other README files)